.cruft.json

```diff
@@ -0,0 +1,31 @@
+{
+  "template": "https://github.com/OpenAstronomy/packaging-guide",
+  "commit": "ad6da17e1d8fd96456566abed30b510bf8b5f8c8",
+  "checkout": null,
+  "context": {
+    "cookiecutter": {
+      "package_name": "MagIEpy",
+      "module_name": "magiepy",
+      "short_description": "Python utilities for the Magnetometer Network of Ireland (MagIE)",
+      "author_name": "DIAS Solar",
+      "author_email": "",
+      "project_url": "magie.ie",
+      "license": "BSD 3-Clause",
+      "minimum_python_version": "3.9",
```

Comment:
I think just at end of py3.10 support might want to move to 3.10 (https://scientific-python.org/specs/spec-0000/)

Reply:
Yes, that's probably a good plan.

Reply:
I have attempted to change everything to 3.10, if you could give a quick sweep that would be appreciated